### PR TITLE
#8: Applied BEM to my project

### PR DESCRIPTION
**Pull request by sfregolina**
I applied BEM to all my files: I changed class names.

### CODE REVIEW

File: src/sass/style.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import "mixins", "navigation", "footer";
+@import "mixins", "navigation", "main", "footer";
```
Comment (Flamov) [inline src/sass/style.scss]: I never knew you could do multiple imports in one line! TIL, thanks!

File: src/sass/main.scss [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+    list-style: none;
+    font-size: .9rem;
+    line-height: 1.35rem;
+  }
+}
+  
+.center {
+  width: 100%;
+  text-align: center;
+}
```
Comment (Flamov) [inline src/sass/main.scss]: Let's add a new line at the end of the file, it should come up in the code linting

File: src/sass/main.scss [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+      .product-select {
+        min-width: 101px;
+        position: relative;
+  
+        &:after {
+          content: "";
+          position: absolute;
+          right: 6px;
+          top: 12px;
+          z-index: -1;
```
Comment (Flamov) [inline src/sass/main.scss]: We'll need to look at this in more detail but I don't suggest using negative values for z-index – typically I'd recommend multiples of 10, but it's down to personal preference

File: src/sass/main.scss [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+    margin: auto;
+  }
+  
+  &__center {
+    max-width: 480px;
+  }
+}
+  
+.zoom img:hover {
+  cursor: url("../images/zoom_in.png"),auto;
```
Comment (Flamov) [inline src/sass/main.scss]: You can use a native value for the `cursor` property so you don't have to use your own image, see here: https://developer.mozilla.org/en-US/docs/Web/CSS/cursor#Syntax

File: src/sass/main.scss [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+main {
+  margin-top: 40px;
+}
+  
+.product-row {
+  width: 100%;
+  padding: 0 30px;
+  overflow: hidden;
+  
+  @include border-box ();
```
Comment (Flamov) [inline src/sass/main.scss]: You don't need to define the mixins as functions or invoke them as such if they don't have arguments/parameters, see the examples here: https://sass-lang.com/documentation/at-rules/mixin

File: src/sass/main.scss [hunk 6]
```diff
@@ -71,7 +71,7 @@ main {
 }
   
 .zoom img:hover {
-  cursor: url("../images/zoom_in.png"),auto;
```
Comment (Flamov) [inline src/sass/main.scss]: Let's make sure we remove the image file from `/images/` too since we're not using it anymore

Comment (sfregolina) [reply src/sass/main.scss]: This is done :)

**Pull request merged**